docs/Displaying a User Interface/Main Menu/index.md

```diff
@@ -43,11 +43,13 @@ The best way to create and update your menu is to the use the Screen Manager API
 To find out more information on how to create `voiceCommands` see the [related documentation](Speech and Audio/Setting Up Voice Commands).
 !@
 
+Newer head units (RPC v7.1+) supports the implementation of secondaryText, tertiaryText and secondaryArtwork which gives the user a richer experience when there is a need to display more data. This new feature is shown [above](#menu-template).
+
 @![iOS]
 ##### Objective-C
 ```objc
 // Create the menu cell
-SDLMenuCell *cell = [[SDLMenuCell alloc] initWithTitle:<#NSString#> icon:<#SDLArtwork#> voiceCommands:<#@[NSString]#> handler:^(SDLTriggerSource  _Nonnull triggerSource) {
+SDLMenuCell *cell = [[SDLMenuCell alloc] initWithTitle:<#NSString#> icon:<#SDLArtwork#> voiceCommands:<#@[NSString]#> secondaryText:<#(nullable NSString *)#> tertiaryText:<#(nullable NSString *)#> secondaryArtwork:<#(nullable SDLArtwork *)#> handler:^(SDLTriggerSource  _Nonnull triggerSource) {
```

Comment:
After reading this initializer, I think that it should be changed in the library to:

```objc
- (instancetype)initWithTitle: secondaryText: tertiaryText: icon: secondaryArtwork: voiceCommands: handler:
```